bloges56/DateSim
# added claire sprite, arcade machine sprites, and edited UI

## Description

Cosmetic changes for the arcade scene which includes adding Claire NPC, sprites for the arcade machines, and updating the UI to look nicer.

## To Test
- Simply test to make sure everything functions as before
- The Claire NPC is not setup yet, so do not worry that interacting with her does not work yet

## Checklist
- [x] Does everything function as before?
